### PR TITLE
Wallet info mask state (EXPOSUREAPP-13800)

### PR DESCRIPTION
**Pull request by chiljamgossow**
new fields for mask state

mock server generates mask state, no dummy data needed

https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-13800

### CODE REVIEW

File: Corona-Warn-App/src/test/java/de/rki/coronawarnapp/ccl/dccwalletinfo/model/DccWalletInfoData.kt [hunk 1]
```diff
@@ -217,6 +219,32 @@ private val certificateReissuance = CertificateReissuance(
...
+        parameters = listOf()
+    ),
+    subtitleText = SingleText(
+        type = "string",
+        localizedText = mapOf("de" to "Für Sie gilt keine Maskenpflicht"),
+        parameters = listOf()
+    ),
+    longText = SingleText(
+        type = "string",
+        localizedText = mapOf("de" to "Aufgrund Ihre Zertifikate [...]"),
```
Comment (MikeMcC399) [inline Corona-Warn-App/src/test/java/de/rki/coronawarnapp/ccl/dccwalletinfo/model/DccWalletInfoData.kt:241]: Ihre => Ihrer (Genitiv Plural)

File: Corona-Warn-App/src/test/resources/ccl/dcc_wallet_info_output_with_mask_state.json [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+      "localizedText": {
+        "de": "Für Sie gilt keine Maskenpflicht"
+      },
+      "parameters": [
+      ]
+    },
+    "longText": {
+      "type": "string",
+      "localizedText": {
+        "de": "Aufgrund Ihre Zertifikate [...]"
```
Comment (MikeMcC399) [inline Corona-Warn-App/src/test/resources/ccl/dcc_wallet_info_output_with_mask_state.json:155]: Ihre => Ihrer

**Review by MikeMcC399 — CHANGES_REQUESTED** [review]
Suggest to check grammar of text (see in-line comments).

**Review by mtwalli — APPROVED** [review]
Lgtm

**Pull request merged**